crates/matrix-sdk-crypto/src/verification/qrcode.rs

```diff
@@ -328,6 +328,7 @@ impl QrVerification {
 
     /// Confirm that the other side has scanned our QR code.
     pub fn confirm_scanning(&self) -> Option<OutgoingVerificationRequest> {
+        info!("User confirmed other side scanned our QR code");
```

Comment:
Can this be a debug or trace instead? Otherwise the verification will cause many many info-level statements

Reply:
yeah will make this one debug.

Reply:
d7f7ad528f0180646473ab477cf7d75af764ce4a